pkg/config/server_config.go

```diff
@@ -172,7 +172,7 @@ func (c *ServerConfig) Validate() error {
 		return fmt.Errorf("failed to validate issue API configuration: %w", err)
 	}
 
-	if c.MinRealmsForSystemStatistics < 2 {
+	if c.MinRealmsForSystemStatistics < 1 {
```

Comment:
Need to update the error. You can cheat and inline with:

```go
if min := 1; c.MinRealmsForSystemStatistics < min {
  return fmt.Errorf("MIN_REALMS_FOR_SYSTEM_STATS cannot be set lower than %d", min)
}
```

Reply:
debug code that didn't get reverted